### ISSUE TITLE
Reference DM in SotBE, and possibly vice-versa

### ISSUE THREAD

**Issue by max-torch** [wesnoth/wesnoth#6174]
There is a good opportunity to reference Delfador's Memoirs in Son of the Black Eye, and possibly vice-versa. Look at these lines from DM:
> But there is something else those orcs said that deeply worries me in a different way. I have wondered why the orcs have been attacking in concert all along the frontier; they do not coordinate well. If that orc was telling the truth and a paramount chief strong enough to marshal their entire host has arisen among them, the King may be walking into a lethal trap at the Ford of Abez.

> While Delfador was still at the Academy, major events were taking place throughout Wesnoth. After centuries of squabbling, a warlord emerged to unite the orcs. This was ill tidings for humans and elves alike.

> Kalenz and Chantal and their troops departed to ride against the remnants of the orcish **Great Horde**. Delfador returned to Weldyn with the King and his gift from the elves, the Book of Crelanu. However, despite the victory, Delfador was deeply worried by the undead menace. On the way back they met with Lionel who was arriving with reinforcements, and were deeply saddened to learn that Leollyn had died under very suspicious circumstances.

It says "After centuries of squabbling". At the same time, in DM's time, Wesnoth and the orcs have only been on the continent for centuries. I believe the Great Horde we are witnessing in DM is the first Great Horde. (There is a campaign opportunity here for the willing: a Northerners campaign about forming the first orcish great horde where the protagonist is Zorlan, by the way he has a nice custom portrait).

In DM. we need some dialog where the orcs themselves formally refer to their collective as "the Great Horde" and possibly even explain how orcish shamans came to be or how the shamans have authority and fit into the equation. Perhaps the shamans are the ones that remember the Ruby of Fire that Haldric brought, and the (false) information that the elves have the ruby. And maybe shamans also know about some other important lore about how to persuade trolls to fight with orcs, and how to conquer wesnoth.

In Son of the Black Eye, as Kapoue himself forms the Great Horde once more, we can make reference to the very first Great Horde that was formed by Zorlan during the time of King Garard II that failed in its attempt to conquer Wesnoth.

Thoughts?

**Comment by max-torch** [comment]
> I know that the campaigns _Delfador Memoirs_, _Eastern Invasion_ and _AOI_ have reworks incoming in the next year or so. Not sure if DM should be referenced SotBE before the rework is done. Perhaps Yumi can weight in?

If not a direct reference, then a dialog connection.

**Comment by Wedge009** [comment]
I feel like this kind of general, non-technical discussion can also be directed to the Wesnoth forums for a wider audience.

Pinging @nemaara as per knyghtmare's mention regarding campaign story revisions.

**Comment by knyghtmare** [comment]
@Dalas121 Should this be closed as well? (#9726)

**Comment by Dalas121** [comment]
Yeah, seems reasonable to close this. Might be worth looking at adding something like this to TDG in the future, but TDG/DM are so different that I don't think an issue for one will directly translate to the other.
